crates/core/src/rpc/minimal.rs

```diff
@@ -23,6 +27,14 @@ use solana_sdk::{
 pub trait Minimal {
     type Metadata;
 
+    #[rpc(meta, name = "getAccountInfo")]
```

Comment:
Instead of introducing this endpoint in `Minimal`, I'd be in favor of introducing the `AccountsData` trait, responsible for grouping a few other methods (that we can left unimplemented for now).
These traits are coming from the canonical implementation, sticking to these architecture could help with maintenance. 

`AccountsData` introduced here: https://github.com/txtx/surfpool/pull/6/files